etc/rh-license-names.json

```diff
@@ -203,6 +203,20 @@
       "http://www.eclipse.org/org/documents/epl-v10.php"
     ]
   },
+  {
+    "name": "Eclipse Public License, Version 2.0",
+    "url": "http://www.eclipse.org/legal/epl-v20.html",
+    "aliases": [
+      "Eclipse Public License, Version 2.0",
+      "Eclipse Public License 2.0",
+      "Eclipse Public License v2.0",
+      "Eclipse Public License - v 2.0",
```

Comment:
"Eclipse Public License - v 2.0" should go to the "name" and the "Eclipse Public License, Version 2.0" should go here. The idea of this override is whenever any of these aliases is found to replace them with the value in "name". Values in "name" and "url" have to be found in http://git.app.eng.bos.redhat.com/git/jboss-licenses.git/plain/rh_license_list.json

Reply:
Great, thanks! How about now?